spec/controllers/idv/agreement_controller_spec.rb

```diff
@@ -9,8 +11,8 @@
 
   before do
     stub_sign_in(user)
+    stub_up_to(:welcome, idv_session: subject.idv_session)
```

Comment:
Would it make sense to have this method be `stub_steps_before` so that you don't have to track that it's "welcome" before agreement? So that this could read:

```suggestion
    stub_steps_before(:agreement, idv_session: subject.idv_session)
```

Reply:
I agree that this would be easier to add and easier to read, but how would it behave for `stub_steps_before(:phone, , idv_session: subject.idv_session)`, which could come from either `:verify_info` or `:ipp_verify_info`?

Reply:
I thought about this some more. Maybe we could provide `stub_steps_before` as an additional interface that hides the knowledge of previous steps, and people can use `stub_up_to` where they need to be more specific.

Reply:
I tried it out in cc2b4bb27876b22a771aaafde47f4b35cec1a5d4. It seems to me that it hides information that might be useful to have in each spec. What do you think?

Reply:
I agree. Leave as-is